include/deal.II/base/vectorization.h

```diff
@@ -224,6 +224,19 @@ class VectorizedArray
   // POD means that there should be no user-defined constructors, destructors
   // and copy functions (the standard is somewhat relaxed in C++2011, though).
 
+  VectorizedArray() = default;
+
+  VectorizedArray(const Number scalar)
```

Comment:
Can you please add a short documentation - "Construct an array with the given scalar broadcast to all lanes."?

Reply:
I have added the documentation. I'll wait with the documentation of `broadcast_from_scalar` until we have decides, if we actually want it.